manifests/p/Prey/Prey/.1.9.15/Prey.Prey.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Prey.Prey
+PackageVersion: .1.9.15
```

Issue with version matching
```suggestion
PackageVersion: 1.9.15
```